### DISCUSSION TITLE
Custom dialogue

### DISCUSSION THREAD

**Discussion by Ajatuks** [Luke100000/minecraft-comes-alive#502] [Q&A]
Hello, i'm new to mods in general and i was wondering if it's possible to make custom dialogue with a villager. If not, is there a different mod compatible with MCA that allows me to do that?

**Comment by Luke100000** [comment]
Yes, but the wiki is quite scarce on it as I thought noone would care about it ^^
https://github.com/Luke100000/minecraft-comes-alive/wiki/Dialogues
I will improve that and throw in an example datapack asap.

**Comment by Luke100000** [comment]
I updated the wiki and added an example datapack/resource pack
